helm/hpcc/templates/_helpers.tpl

```diff
@@ -2264,21 +2262,21 @@ spec:
         selector: {{ $metric.selector }}
       {{- end }}
      {{- else if eq $metric.type "Object" }}
-     {{- if hasKey $metric "describedObject" }}
+      {{- if hasKey $metric "describedObject" }}
       metric:
         name: {{ $metric.name }}
       describedObject:
         apiVersion: {{ $metric.describedObject.apiVersion }}
         kind: {{ $metric.describedObject.kind }}
         name: {{ $metric.describedObject.name }}
-     {{- end }}
+      {{- end }}
      {{- else if eq $metric.type "External"}}
       metric:
         name: {{ $metric.name }}
       {{- if hasKey $metric "selector" }}
         selector:
           matchLabels:
-            {{ toYaml $metric.selector.matchLabels }}
+{{ toYaml $metric.selector.matchLabels | indent 12}}
```

Comment:
minor: Better style is to indent as before, have a leading - and use nindent.
```
            {{- toYaml $metric.selector.matchLabels | nindent 12 }}
```

Reply:
these are expensive style changes which amount to no-ops... will make change as suggested anyhow

Reply:
Suggested change applied, change now adheres to source code standard, and results in appropriately indented helm values:

```# Source: hpcc/templates/roxie.yaml
---
apiVersion: autoscaling/v2
kind: HorizontalPodAutoscaler
metadata:
  name: roxie-hpa
spec:
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: roxie
  minReplicas: 3
  maxReplicas: 15
  metrics:
  - type: Resource
    resource:
      name: cpu
      target:
        type: Utilization
        averageUtilization: 50
  - type: Pods
    pods:
      metric:
        name: packets-per-second
      target:
        type: AverageValue
        averageValue: 1k
  - type: External
    external:
      metric:
        name: queue_messages_ready
        selector:
          matchLabels:
            queue: worker_tasks
      target:
        type: AverageValue
        averageValue: 30
  - type: Object
    object:
      metric:
        name: requests-per-second
      describedObject:
        apiVersion: api/v1
        kind: Pod
        name: certain-pod-of-service-B
      target:
        type: AverageValue
        averageValue: 30
---